pkg/operator/bootstrap.go

```diff
@@ -169,6 +169,26 @@ func RenderBootstrap(
 		name:     "manifests/machineconfigserver/kube-apiserver-serving-ca-configmap.yaml",
 		filename: "manifests/kube-apiserver-serving-ca-configmap.yaml",
 	}}
+
+	if infra.Status.PlatformStatus.BareMetal != nil {
+		manifests = append(manifests, []struct {
+			name     string
+			data     []byte
+			filename string
+		}{{
+			name:     "manifests/baremetal/coredns.yaml",
+			filename: "baremetal/manifests/coredns.yaml",
+		}, {
+			name:     "manifests/baremetal/coredns-corefile.tmpl",
+			filename: "baremetal/static-pod-resources/coredns/Corefile.tmpl",
+		}, {
+			name:     "manifests/baremetal/keepalived.yaml",
+			filename: "baremetal/manifests/keepalived.yaml",
+		}, {
+			name:     "manifests/baremetal/keepalived.conf.tmpl",
+			filename: "baremetal/static-pod-resources/keepalived/keepalived.conf.tmpl",
```

Comment:
shouldn't all these end up under `bootstrap/` similar to https://github.com/openshift/machine-config-operator/pull/1002/files#diff-d6f38878757202a810cfa8038e7ae122R161

Have you tested this out manually to verify this is applied at bootstrap?

Reply:
I believe @celebdor made this change because having them in bootstrap caused them to be applied twice - once by kubelet and again during cluster-bootstrap. We will be explicitly copying them into place in the installer[0], so the specific location here doesn't matter much.

0: https://github.com/celebdor/installer/commit/134d74cdab091303721b00d2111330c862420b23